README.md

```diff
@@ -93,22 +91,19 @@ function mysum(list)
 end
 )
 
-plot(code, fontsize=5, shorten=0.2, axis_buffer=0.05)
+plot(code, fontsize=5, shorten=0.2, axis_buffer=0.15)
 
 ```
 
-![graph_four](https://user-images.githubusercontent.com/2822757/49310588-faa78600-f4de-11e8-95cf-4587d0ba1077.png)
+![](assets/AST_example.png)
 
 #### Julia Type Trees
 
 ```julia
 using GraphRecipes
 using Plots
 
-pyplot(size=(800,600))
```

Comment:
If you still want to set default values, you can replace `pyplot(` with `default(`

Reply:
I have generated a PDF of the README locally and it looks good to me. We will make some more aesthetic changes once marker handling has been improved.